docs/en/index.rst

```diff
@@ -101,7 +101,7 @@ The ``FilesystemCache`` driver stores the cache data on the local filesystem.
 .. code-block:: php
     $cache = new FilesystemCache('/path/to/cache/directory');
 
-MemecacheCache
+MemcacheCache
 --------------
```

Comment:
Can you delete one dash to ensure consistent length?

Reply:
No worries. It’s fascinating that this PR has the same dash alignment issue as the other one 😂